tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -486,6 +493,7 @@ class GFF3Importer extends TripalImporter {
 
     // Set the private member variables of this class using the loader inputs.
     $this->organism_id = $arguments['organism_id'];
+    $this->update_names=[];
```

Comment:
This line isn't necessary because this variable is set to an empty array on line 112

Reply:
Fixed.